docs/source/governance.rst

```diff
@@ -0,0 +1,21 @@
+.. _governance:
+
+Flash Governance | Persons of interest
+======================================
+
+Leads
+-----
+- William Falcon (`williamFalcon <https://github.com/williamFalcon>`_)
+- Thomas Chaton (`tchaton <https://github.com/tchaton>`_)
+- Ethan Harris (`ethanwharris <https://github.com/ethanwharris>`_)
+- Jirka Borovec (`Borda <https://github.com/Borda>`_)
+- Kaushik Bokka (`kaushikb11 <https://github.com/kaushikb11>`_)
+- Justus Schock (`justusschock <https://github.com/justusschock>`_)
+- Carlos Mocholí (`carmocca <https://github.com/carmocca>`_)
+- Sean Narenthiran (`SeanNaren <https://github.com/SeanNaren>`_)
```

Comment:
this is a copy from PL, so are also all leads from PL leads in Flash?

Reply:
Not quite copied, adapted :smiley: These are all the PL leads who have also made contributions to flash

Reply:
yes, but for example I not feel like lead for Flash, rather be just core :P